### ISSUE TITLE
Possible to trigger when a new Container is added to storage account?

### ISSUE THREAD

**Issue by dennismulder** [Azure/azure-webjobs-sdk#572]
Is it possible to trigger when a new container is added to storage?

All documentation which is available online is not mentioning this scenario. The [Blobtrigger] decoration should look something like this below and should be able to fire when a new container and some new files are added. But somehow it doesn’t. The question is: is this possible at all?
 
```c#
public static void DoSomething(
        [BlobTrigger("$root/{name}/{file}.{ext}")] Stream incoming, string name,string file,string ext, TextWriter log)
        {
            log.WriteLine(string.Format("name: {0", name));
            log.WriteLine(string.Format("file: {0", file));
            log.WriteLine(string.Format("ext: {0", ext));
        }
 ```

And if this is possible, how would the match pattern look like?

**Comment by mathewc** [comment]
Not possible. All the various listeners/triggers we have only operate on existing container types (e.g. Queues/Blobs). They only trigger when something is added to them, not when new containers/queues are added.

**Comment by dennismulder** [comment]
Thanks for the quick response! Can this be considered as a feature request?

**Comment by mathewc** [comment]
Yes we'll treat it as a feature request, and we'll discuss this in our issue triage :) We'll add any results to this issue. Thanks.

**Comment by rustd** [comment]
Thank you for this suggestion We will consider this for a future release.
